pkg/util/utils_test.go

```diff
@@ -17,3 +18,45 @@ func TestStringInSlice(t *testing.T) {
 	// string is not in empty slice
 	assert.False(t, StringInSlice("one", []string{}))
 }
+
+func TestParseChanges(t *testing.T) {
+	// CMD=/bin/sh
+	_, vals, err := ParseChanges("CMD=/bin/sh")
+	assert.EqualValues(t, []string{"/bin/sh"}, vals)
+	assert.NoError(t, err)
+	// CMD [/bin/sh]
+	_, vals, err = ParseChanges("CMD [/bin/sh]")
+	assert.EqualValues(t, []string{"/bin/sh"}, vals)
+	assert.NoError(t, err)
+
+	// CMD ["/bin/sh"]
+	_, vals, err = ParseChanges(`CMD ["/bin/sh"]`)
+	assert.EqualValues(t, []string{`"/bin/sh"`}, vals)
+	assert.NoError(t, err)
+
+	// CMD ["/bin/sh", "-c", "ls"]
+	_, vals, err = ParseChanges(`CMD ["/bin/sh","c","ls"]`)
+	assert.EqualValues(t, []string{`"/bin/sh"`, `"c"`, `"ls"`}, vals)
+	assert.NoError(t, err)
+
+	// CMD "/bin/sh"]
+	_, _, err = ParseChanges(`CMD "/bin/sh"]`)
+	assert.Equal(t, `invalid value "/bin/sh"]`, err.Error())
+
+	// CMD [bin/sh
+	_, _, err = ParseChanges(`CMD "/bin/sh"]`)
+	assert.Equal(t, `invalid value "/bin/sh"]`, err.Error())
+
+	// CMD ["/bin /sh"]
+	_, _, err = ParseChanges(`CMD ["/bin /sh"]`)
+	assert.Equal(t, `invalid value "/bin /sh"`, err.Error())
+
+	// CMD?
+	_, _, err = ParseChanges(`CMD?`)
+	assert.Equal(t, `invalid format CMD?`, err.Error())
+
+	// empty values for CMD
+	_, _, err = ParseChanges(`CMD `)
+	assert.Equal(t, `invalid value `, err.Error())
+
```

Comment:
Would be nice to have a test for `--change LABEL=blue=image` here and probably my question case `--change LABEL = blue=image`

Reply:
I added the testcase for this.